### DISCUSSION TITLE
Not getting Dpd retransmission when backhaul is disrupted

### DISCUSSION THREAD

**Discussion by karthikadiga123** [strongswan/strongswan#828] [General]
Hi All, I'm using strongswan 5.9.1 vpn client. When backhaul is disconnected(dropped packets from/to server through iptables) strongswan client is not recognising it and not sending dpd retransmission. May I know what needs to be done for this? Any configurations to be added?

**Comment by tobiasbrunner** [comment]
Refer to the documentation of the `dpd_*` options (e.g. [here](https://wiki.strongswan.org/projects/strongswan/wiki/swanctlconf)).
